dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp

```diff
@@ -232,14 +279,39 @@ PrehandleResult KVStore::preHandleSSTsToDTFiles(
                     new_region->toString(true));
                 break;
             }
+            else if (result.error == ReadFromStreamError::Aborted)
+            {
+                LOG_INFO(
+                    log,
+                    "Apply snapshot is aborted, cancelling. region_id={} term={} index={}",
+                    region_id,
+                    term,
+                    index);
+            }
+
+            (void)table_drop_lock; // the table should not be dropped during ingesting file
+            break;
+        }
+        catch (DB::Exception & e)
+        {
+            if (e.code() == ErrorCodes::TABLE_IS_DROPPED)
```

Comment:
The `TABLE_IS_DROPPED` exception here can still be thrown by some `AtomicGetStorageSchema`.